.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: ci-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository info'
+        required: true
+      branch:
+        description: 'Name of the branch'
+        required: true
+      target_repository:
+        description: 'Repository name of the terratest'
+        required: true
+      target_pr:
+        description: 'PR number on the terratest repo'
+        required: true
+
+jobs:
```

Comment:
It looks like the job is running the linter for `modules/azure`, but not the tests. Is that intentional?

Reply:
actually, I separated steps to run linter for `modules/azure` and `test/azure` folders.

[lint modules/azure folder](https://github.com/gruntwork-io/terratest/pull/594/files#diff-e9f950f17198d3d5e3122a44230a09b9R31)

[lint test/azure folder](https://github.com/gruntwork-io/terratest/pull/594/files#diff-e9f950f17198d3d5e3122a44230a09b9R40)

Reply:
Oh sorry, what I meant was that you weren't running the tests in `modules/azure` folder, and not that you weren't running the linter in tests folder.

If you look in [modules/azure](https://github.com/gruntwork-io/terratest/tree/master/modules/azure), you will find a few unit test files in there and I was wondering if we were intentionally going to remove those or not.

Reply:
@polatengin we should run go test for unit test also.  Can we run those test first?

Reply:
I spoke to @polatengin  and we are moving the test in modules to the test folder.  They are actually integration test.  We can discuss unit test approach in future PR.  He will add a commit that moves those test files. @yorinasub17  does that work?

Reply:
Yup that is fine with me! Just wanted to make sure how we wanted to do that going forward.

Reply:
I created another PR for that, it seemed not related to CI pipeline to me.

here is the PR : https://github.com/gruntwork-io/terratest/pull/596